### PR TITLE
PR for #3: Regression on years greater than 1960 

### LINKED ISSUE #3: Running regression on years greater or equal to 1960
The goal of the issue is to modify code in the analysis folder [~/analysis/code/analyze_data.py](https://github.com/shrishj/template/blob/6ead6b59455a03c5dd6cfc066adcf459a060f091/analysis/code/analyze_data.py) to run an additional regression on years greater or equal to 1960 and export the model coefficient, standard errors and p-value into a new table output found [analysis/output/regression.csv](https://github.com/shrishj/template/blob/6ead6b59455a03c5dd6cfc066adcf459a060f091/analysis/output/regression.csv). This follows from the [Practice Task Instructions](https://github.com/gslab-econ/lab-manual/wiki/Practice-Task).

cc @ShiqiYang2022 @jc-cisneros @snairdesai

### PR DESCRIPTION
**Pull request by shrishj**
I have modified code in the analysis folder [~/analysis/code/analyze_data.py](https://github.com/shrishj/template/blob/54ee1328331ab773e55f5ae922870c099ce12e57/analysis/code/analyze_data.py) so that we create a new table to hold our values in [analysis/output/regression_subset.csv](https://github.com/shrishj/template/blob/54ee1328331ab773e55f5ae922870c099ce12e57/analysis/output/regression_subset.csv). This follows from the [Practice Task Instructions](https://github.com/gslab-econ/lab-manual/wiki/Practice-Task).

I have assigned @ShiqiYang2022, @jc-cisneros, and @snairdesai to review this request.

Closes #3

### CODE REVIEW

File: data/output/chips_sold.pdf
Comment (ShiqiYang2022) [inline data/output/chips_sold.pdf:1]: @shrishj I think you should also add code to include the table into this `.pdf`, thanks!

**Review by ShiqiYang2022 — CHANGES_REQUESTED** [review]
Thanks @shrishj! I have some requests for you to change the code according.

File: paper_slides/output/paper.pdf
Comment (ShiqiYang2022) [inline paper_slides/output/paper.pdf:1]: @shrishj I think you should also add code to include the table into this `.pdf`, thanks!

Comment (shrishj) [reply paper_slides/output/paper.pdf:1]: Hi @ShiqiYang2022 , here is my changed version: [1675c2d](https://github.com/shrishj/template/pull/4/commits/1675c2dea78fa321cbb42c30f9399ee98e1fcf3f)

**Review by ShiqiYang2022 — APPROVED** [review]
I think the change looks good so I am approving. It could be better if we modify the name of the table according, but this is trivial. Thanks for the work @shrishj!

**Pull request merged**